content/en/docs/concepts/services-networking/service.md

```diff
@@ -5,7 +5,7 @@ title: Service
 feature:
   title: Service discovery and load balancing
   description: >
-    No need to modify your application to use an unfamiliar service discovery mechanism. Kubernetes gives pods their own IP addresses and a single DNS name for a set of pods, and can load-balance across them.
+    No need to modify your application to use an unfamiliar Service discovery mechanism. Kubernetes gives Pods their own IP addresses and a single DNS name for a set of Pods, and can load-balance across them.
```

Comment:
I think “service discovery” here is the wider sense, rather than talking about Service in the Kubernetes sense.